exercises/gigasecond/introduction.md

```diff
@@ -0,0 +1,20 @@
+# Introduction
+
+The way we measure time is kind of messy.
+We have 60 seconds in a minute, and 60 minutes in an hour.
+This comes from ancient Babylon, where they used 60 as the basis for their number system.
+We have 24 hours in a day, 7 days in a week, and months?
+Well, for months it depends not only on which month it is, but also what calendar is used in the country you live in.
+
+If we ever colonize Mars or some other planet, it's going to get even messier.
+If someone says "year" do they mean Earth-year or Mars-year?
```

Comment:
Probably too much, but I love this article on the consortium that is trying to decide [time standards for the moon](https://www.nature.com/articles/d41586-023-00185-z).  Turns out that clocks tick at different speeds due to different gravitational fields...

Reply:
Ok, this is fantastic.

My sense is that it's on the "too much" side of the line, but I love it.